### PR TITLE
Update to cli 3.1.3

### PR DESCRIPTION
**Pull request by facumartin**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+    def _fetch_existing_mappings(self) -> None:
+        self._mappings["SiteReader"]: List[
+            SiteReader
+        ] = self.custom_types.SiteReader.list()
+        self._mappings["InverterReader"]: List[
+            SiteReader
+        ] = self.custom_types.InverterReader.list()
+
+        print(self._mappings)
```
Comment (marianogaragiola) [inline main.py:49]: Do not use print, use the splight logger, this way the logs will be handled by the Log Streamer

Comment (facumartin) [reply main.py:49]: sorry, I forgot to delete it, that shouldn't be there.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+            "asset_name": new_site_attribute.asset.name,
+            "attribute_id": new_site_attribute.attribute.id,
+            "attribute_name": new_site_attribute.attribute.name,
+            "resource": new_site_attribute.resource,
+        }
+        return _dict
+
+    def log_site_attributes(self) -> None:
+        for attribute in self.site_attributes:
+            logger.info(
```
Comment (marianogaragiola) [inline main.py:280]: We need to change these log messages, in the current state the messages have several lines, that could break the log pipeline in Elastic Search, we need the message to be one line only. Also, if you take a look at the message we can see debug information, so I would make some changes here in order

File: spec.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "tags": [
         "connector"
     ],
-    "splight_cli_version": "2.3.19",
+    "splight_cli_version": "3.1.3",
```
Comment (marianogaragiola) [inline spec.json:7]: Update the component version too, the new version should be "2.0.0"

**Review by marianogaragiola — APPROVED** [review]

**Pull request merged**